cmd/vulcan-zap/manifest.toml

```diff
@@ -6,4 +6,14 @@ Timeout = 36000 # 10 hours. Expressed in seconds as an integer.
 # 10003 - Vulnerable JS Library - Duplicates the Retire.js check.
 # 10108 - Reverse Tabnabbing - Not relevant for modern browser versions.
 # Source: https://www.zaproxy.org/docs/alerts/
-Options = '{"depth": 2, "active": true, "min_score": 0, "disabled_scanners": ["10062", "10003", "10108"]}'
+Options = """{
+    "depth": 2, 
+    "active": true, 
+    "min_score": 0, 
+    "disabled_scanners": ["10062", "10003", "10108"],
+    "max_spider_duration": 0,
+    "max_scan_duration": 0, 
+    "max_rule_duration": 0,
```

Comment:
Should we set some defaults congruent with the default check timeout of 10 h, i.e. max_scan_duration: 500 to obtain the results before the check is timed out?

Reply:
In my opinion, until we don't define a convention within the team (if to indicate all the available options in the manifest or not) you can leave the options with the "most convenient" default values.

Just to confirm, `0` value for `duration` parameters means infinite, right ?

If that is the case, I won't change `duration` options to match check timeout in order to keep the same behaviour we have at the moment.

Reply:
Makes sense, thanks